src/common/pins.h

```diff
@@ -93,6 +93,11 @@
 #define SAMWISE_UART_TX (31)
 #define SAMWISE_UART_RX (30)
 
+#define SAMWISE_UART_TX_TO_ADCS (24)
+#define SAMWISE_UART_RX_FROM_ADCS (25)
+#define SAMWISE_ADCS_EN (26)
+#define SAMWISE_ADCS_UART (uart1)
+
```

Comment:
for successful build checks, we will need mocked pins in this file for the PICO board as well (although we won't use nor read from them).